test/truncate.test.js

```diff
@@ -0,0 +1,13 @@
+const { truncate } = require("../lib");
+
+describe("truncate(s,n)", () => {
+  it("returns truncated string", () => {
+    expect(truncate("hello", 3)).toBe("hell");
+    expect(truncate("pink panther", 3)).toBe("pink");
+  });
+  it("returns an empty array when the inputs are invalid", () => {
+    expect(truncate("", 2)).toBe("");
+    expect(truncate(324, 2)).toBe("");
+    expect(truncate("negative", -2)).toBe("");
+  });
+});
```

Comment:
Were you going to write a skeleton function for `truncate`?

Reply:
Yeah, I wanted to do so, but realized it's a bit of an over kill cus you can already do that with built in functions...

Reply:
Ok. Lets remove these tests then.